symfony/phpunit-bridge/5.3/bin/phpunit

```diff
@@ -1,16 +1,19 @@
 #!/usr/bin/env php
 <?php
-if (file_exists(dirname(__DIR__).'/vendor/phpunit/phpunit/phpunit')) {
-    require dirname(__DIR__).'/vendor/phpunit/phpunit/phpunit';
+
+if (!ini_get('date.timezone')) {
+    ini_set('date.timezone', 'UTC');
+}
+
+if (is_file(dirname(__DIR__).'/vendor/phpunit/phpunit/phpunit')) {
+    define('PHPUNIT_COMPOSER_INSTALL', dirname(__DIR__).'/vendor/autoload.php');
+    require PHPUNIT_COMPOSER_INSTALL;
+    PHPUnit\TextUI\Command::main();
```

Comment:
Is this the real significant portion of this PR? Could you explain a bit further?

Reply:
This is the only possible fix for #936

Reply:
I see how there is no PHP file in phpunit to directly require. And so, you've mimicked their phpunit executable.